src/renderer/containers/UploadSelectionPage/DragAndDropPrompt/styles.pcss

```diff
@@ -4,7 +4,7 @@
   display: flex;
   flex-direction: column;
   margin-bottom: 20px;
-  margin-top: 25px;
+  margin-top: 15px;
```

Comment:
Shortened this to account for the additional space created by the new <div> in `UploadTypeSelector`